### ISSUE TITLE
Error page 

### ISSUE THREAD

**Issue by Voidseption** [Monika-After-Story/MonikaModDev#5355]
I was scrolling though the unseen menu and clicked on a test prog button by mistake, and it brings me to the error page:

[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script-ch30.rpy", line 1496, in script call
  File "game/event-handler.rpy", line 2061, in script call
  File "game/dev/dev_sprites.rpy", line 183, in script
    call screen mas_gen_scrollable_menu(top_level_menu, store.mas_moods.MOOD_AREA, store.mas_moods.MOOD_XALIGN, returner)
  File "renpy/common/000statements.rpy", line 466, in execute_call_screen
    args, kwargs = a.evaluate()
  File "game/dev/dev_sprites.rpy", line 183, in <module>
    call screen mas_gen_scrollable_menu(top_level_menu, store.mas_moods.MOOD_AREA, store.mas_moods.MOOD_XALIGN, returner)
AttributeError: 'StoreModule' object has no attribute 'MOOD_AREA'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script-ch30.rpy", line 1496, in script call
  File "game/event-handler.rpy", line 2061, in script call
  File "game/dev/dev_sprites.rpy", line 183, in script
    call screen mas_gen_scrollable_menu(top_level_menu, store.mas_moods.MOOD_AREA, store.mas_moods.MOOD_XALIGN, returner)
  File "C:\Users\voids\Desktop\Doki Doki Literature Club\renpy\ast.py", line 1706, in execute
    self.call("execute")
  File "C:\Users\voids\Desktop\Doki Doki Literature Club\renpy\ast.py", line 1724, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\voids\Desktop\Doki Doki Literature Club\renpy\statements.py", line 145, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 466, in execute_call_screen
    args, kwargs = a.evaluate()
  File "C:\Users\voids\Desktop\Doki Doki Literature Club\renpy\ast.py", line 184, in evaluate
    args.append(renpy.python.py_eval(v, locals=scope))
  File "C:\Users\voids\Desktop\Doki Doki Literature Club\renpy\python.py", line 1749, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\voids\Desktop\Doki Doki Literature Club\renpy\python.py", line 1743, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/dev/dev_sprites.rpy", line 183, in <module>
    call screen mas_gen_scrollable_menu(top_level_menu, store.mas_moods.MOOD_AREA, store.mas_moods.MOOD_XALIGN, returner)
AttributeError: 'StoreModule' object has no attribute 'MOOD_AREA'

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.10.7
[/code]

i restarted the game but it monika would just say "oh where was i..." and then brings me back here. I clicked ignore, but it would just bring up another error page:

[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script-ch30.rpy", line 1496, in script call
  File "game/event-handler.rpy", line 2061, in script call
  File "game/dev/dev_sprites.rpy", line 201, in script
    python:
  File "game/dev/dev_sprites.rpy", line 211, in <module>
    if sp_obj_to_test.entry_pp is not None:
AttributeError: 'NoneType' object has no attribute 'entry_pp'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script-ch30.rpy", line 1496, in script call
  File "game/event-handler.rpy", line 2061, in script call
  File "game/dev/dev_sprites.rpy", line 201, in script
    python:
  File "C:\Users\voids\Desktop\Doki Doki Literature Club\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\voids\Desktop\Doki Doki Literature Club\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/dev/dev_sprites.rpy", line 211, in <module>
    if sp_obj_to_test.entry_pp is not None:
AttributeError: 'NoneType' object has no attribute 'entry_pp'

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.10.7
[/code]

Then i click ignore again and it brings me back to the previous page, i don't know what to do to make her stop...

**Comment by scpketer** [comment]
Back up your persistent file and reinstall the game with mod. Seems like some dev .rpy files interfere with prebuilt code?

**Comment by Voidseption** [comment]
alright, and how do i do that? (sorry but i'm not like a tech guy)

**Comment by scpketer** [comment]
Here's a guide on how to back up your persistent data.

https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-want-to-back-up-my-persistent

**Comment by Booplicate** [comment]
That doesn't look like something wrong with the persistent. Don't use the dev files.
Should be enough to just reinstall the game and mod, download the game from https://ddlc.moe/ and the mod from http://monikaafterstory.com/.

**Comment by multimokia** [comment]
You installed the source code version of MAS. Delete the `dev` folder in your `\game` folder and all `.rpy` files in there too.

**Comment by Voidseption** [comment]
already done. i redownloaded the game and it all went good, thanks all
